src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
@pavarnos according to my understanding in this case this should be string than array. Tested with https://3v4l.org/WvpMR . Also from twitter https://twitter.com/localheinz/status/815468361463332864 I was looking at http://stackoverflow.com/questions/13787426/how-to-declare-unlimited-parameters-in-docblock/24337184#24337184 .